sc/shared/src/test/scala/sigmastate/TestingInterpreterSpecification.scala

```diff
@@ -202,6 +203,21 @@ class TestingInterpreterSpecification extends CompilerTestingCommons
         |}""".stripMargin)
   }
 
+  property("Evaluate boolean casting ops") {
```

Comment:
This `property` runs the test block for each activated script version / ergotree version. It's currently failing on `v3` script version because [`SBoolean.getMethods`](https://github.com/ScorexFoundation/sigmastate-interpreter/pull/932/files#diff-a26da11ac5b566197ea7da18de380e7f4b893212200c7a96d46be8869837c5a2R916) only seems to get called once at the start of the test suite when script version is 0 so it returns a `Seq()` for all versions.

I'm not sure why `getMethods` isn't called for each test case